### PR TITLE
Added GLFW Timer input device for cross platform frame timing data.

### PR DESCRIPTION
**Pull request by dtorban**
This is necessary to fix part of #90.  We need a separate input device for timing since the VRGLFWInputDevice will be created per process.  We need to specify which process will handle the timing.  If we don't have this, multiple processes will all return conflicting time information.  For example, if we had a cave with 4 walls running on 4 different processes, each wall would return a separate frame elapsed time.  Therefore I added a VRGLFWTimer input device which handles it.  The good thing is that it relies on the Windowing library and not C++11.

**In other words, the test-interactive examples will fail (or look really shaky) in a cave without this fix.**

Here is an example of how to add it to the config file:

```xml
<MinVR>
	<GLFWPlugin pluginType="MinVR_GLFW"/>
        <GLFWTimer inputdeviceType="VRGLFWTimer"/>
         ...
	<VRSetups>
		<Desktop hostType="VRStandAlone">
	           ...
		</Desktop>
	</VRSetups>
</MinVR>
```

### CODE REVIEW

**Comment by tsgouros** [comment]
I thought the goal was to add something that was independent of GLFW? Can that dependency be abstracted away?

**Comment by dtorban** [comment]
Yes, but this is a quick fix.  The current approach will not work.

**Comment by dtorban** [comment]
Long term we can add a platform independent timer.  But we need something that works right now.  This pull request is an easy solution.

**Review by tsgouros — APPROVED** [review]
I don't have the capacity to heck this out, and would prefer that code comments prominently indicate this is just a quick fix that ought to be replaced as soon as is feasible. Even better if those comments would sketch out what a real solution would look like.

**Comment by dtorban** [comment]
I don't think this is a fake solution.  It meets a real need.  Issue #90 is open for discussion for a cross platform timer.

**Comment by dtorban** [comment]
But we can hold off for now :).

**Comment by dtorban** [comment]
Actually, it is pretty much what we have right now, but fixed since it is in its own input device and is not duplicated across all processes.  So this issue and the cross platform issue are essentially different issues.

**Comment by dtorban** [comment]
Oops, thanks Tom, I will add a comment mentioning the long term fix.

**Comment by dtorban** [comment]
Going to fix this with cross platform timer instead.
